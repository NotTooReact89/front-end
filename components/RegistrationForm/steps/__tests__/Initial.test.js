import React from 'react';
import { Formik } from 'formik';
import createSnapshotTest from 'test-utils/createSnapshotTest';

import Initial from '../Initial';

describe('RegistrationForm/Steps/Initial', () => {
  it('should render in context of Formik', () => {
    createSnapshotTest(
      <Formik initialValues={Initial.initialValues} validationSchema={Initial.validationSchema}>
        <Initial />
      </Formik>,
    );
  });
});
