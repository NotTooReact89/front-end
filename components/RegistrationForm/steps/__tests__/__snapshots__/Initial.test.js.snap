// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegistrationForm/Steps/Initial should render in context of Formik 1`] = `
Array [
  <h2
    className="row"
  >
    User Information
  </h2>,
  <div
    className="row"
  >
    <div
      className="field"
    >
      <label
        className="Label"
        htmlFor="email"
      >
        Email*
      </label>
      <div
        className="inputFeedbackGrouping"
      >
        <input
          autoComplete="username email"
          className="Input"
          disabled={false}
          id="email"
          name="email"
          onBlur={[Function]}
          onChange={[Function]}
          type="email"
          value=""
        />
      </div>
    </div>
    <div
      className="field"
    >
      <label
        className="Label"
        htmlFor="confirm-email"
      >
        Confirm Email*
      </label>
      <div
        className="inputFeedbackGrouping"
      >
        <input
          autoComplete="username email"
          className="Input"
          disabled={false}
          id="confirm-email"
          name="confirm-email"
          onBlur={[Function]}
          onChange={[Function]}
          type="email"
          value=""
        />
      </div>
    </div>
  </div>,
  <div
    className="row"
  >
    <div
      className="field"
    >
      <label
        className="Label"
        htmlFor="password"
      >
        Password*
      </label>
      <div
        className="inputFeedbackGrouping"
      >
        <input
          autoComplete="new-password"
          className="Input"
          disabled={false}
          id="password"
          name="password"
          onBlur={[Function]}
          onChange={[Function]}
          type="password"
          value=""
        />
      </div>
    </div>
    <div
      className="field"
    >
      <label
        className="Label"
        htmlFor="confirm-password"
      >
        Confirm Password*
      </label>
      <div
        className="inputFeedbackGrouping"
      >
        <input
          autoComplete="new-password"
          className="Input"
          disabled={false}
          id="confirm-password"
          name="confirm-password"
          onBlur={[Function]}
          onChange={[Function]}
          type="password"
          value=""
        />
      </div>
    </div>
  </div>,
  <div
    className="pulledLeft"
  >
    <div
      className="field"
    >
      <label
        className="Label"
        htmlFor="zipcode"
      >
        Zipcode*
      </label>
      <div
        className="inputFeedbackGrouping"
      >
        <input
          autoComplete="postal-code"
          className="Input"
          disabled={false}
          id="zipcode"
          name="zipcode"
          onBlur={[Function]}
          onChange={[Function]}
          type="text"
          value=""
        />
      </div>
    </div>
  </div>,
]
`;
